app/dashboard/models.py

```diff
@@ -2198,21 +2198,14 @@ def to_dict(self, activities=True, leaderboards=True, network=None, tips=True):
         }
 
         if activities:
-            fulfilled = self.fulfilled.filter(
+            all_activities = self.activities.filter(
                 bounty__network=network
```

Comment:
Can we pipe this together with `self.activites.filter(tip__network=network)` or combine with `Q()` ?